sentry-android-core/src/main/java/io/sentry/android/core/InternalSentrySdk.java

```diff
@@ -193,6 +195,66 @@ public static SentryId captureEnvelope(final @NotNull byte[] envelopeData) {
     return null;
   }
 
+  public static Map<String, Object> getAppStartMeasurement() {
+    final @NotNull AppStartMetrics metrics = AppStartMetrics.getInstance();
+    final @NotNull List<Map<String, Object>> spans = new ArrayList<>();
+
+    final @NotNull TimeSpan processInitNativeSpan = new TimeSpan();
+    processInitNativeSpan.setStartedAt(metrics.getAppStartTimeSpan().getStartUptimeMs());
+    processInitNativeSpan.setStartUnixTimeMs(
+        metrics.getAppStartTimeSpan().getStartTimestampMs()); // This has to go after setStartedAt
+    processInitNativeSpan.setStoppedAt(metrics.getClassLoadedUptimeMs());
+
+    final @NotNull Map<String, Object> processInitSpan = new HashMap<>();
+    processInitSpan.put("description", "Process Initialization");
+    processInitSpan.put("start_timestamp_ms", processInitNativeSpan.getStartTimestampMs());
+    processInitSpan.put("end_timestamp_ms", processInitNativeSpan.getProjectedStopTimestampMs());
+    spans.add(processInitSpan);
+
+    final @NotNull Map<String, Object> applicationOnCreateSpan = new HashMap<>();
```

Comment:
I think you should only accept `TimeSpan` objects which are valid: `timespan.hasStarted() && timespan.hasStopped()`